pkg/clusters/addons/kong/builder.go

```diff
@@ -36,6 +39,18 @@ func (b *Builder) WithPostgreSQL() *Builder {
 	return b
 }
 
+func (b *Builder) WithEnterprise(dbMode DBMode) *Builder {
+	b.dbmode = dbMode
+	b.enterprise = true
+	return b
```

Comment:
Are we forgoing the ability to provide the license? If so can you please help me to better understand the reasoning?

Reply:
No, we keep saying it does not require a license to bump enterprise.

Reply:
Sorry we must not be on the same page: ultimately to effectively use enterprise you need a license. It seems like here is a good place to allow it to be passed in.

Reply:
Shane, you do not need a license to bump enterprise image. Only specific test cases such as Workspace, RBAC (per Travis) request license. Hope this is more clear.

Reply:
Looks like we have solid reasoning to add license handling in this PR to make it possible to test enterprise completely and to avoid disparate mechanisms developing around it, I would like to see this added to this PR please.

Reply:
e02d137c22f069e1af34af42d9e3367a1e7e20b3